packages/owlbot-bootstrapper/common-container/interfaces.ts

```diff
@@ -49,4 +49,59 @@ export interface CliArgs {
 export interface InterContainerVars {
   branchName: string;
   owlbotYamlPath: string;
+  // API short name (e.g., accessapproval)
+  apiShortName: string;
+  // API pretty or full name (e.g., Access Approval API)
+  apiPrettyName?: string;
+  // Product documentation (e.g., https://cloud.google.com/cloud-provider-access-management/access-approval/docs)
+  apiProductDocumentation?: string;
+  // Client documentation (e.g., https://cloud.google.com/nodejs/docs/reference/access-approval/latest)
+  apiClientDocumentation?: string;
+  // API Issue tracker (e.g., https://issuetracker.google.com/issues/new?component=190865&template=1161103)
+  apiIssueTracker?: string;
+  // API Release level or launch stage (e.g., Beta)
+  apiReleaseLevel?: string;
+  // API ID (as described by 'name' in googleapis/google/*/*/*_v*.yaml)
+  // This is confusing notation as we also have apiId which refers to the unique identifier;
+  // But this is how most .repo-metadata.json files have their configuration.
+  apiId?: string;
+}
+
+export interface ApiFields {
```

Comment:
Can you add the path to proto file? For Java, we consider "google/cloud/dataform" (no version at the end) as path to proto.

You may want to add the version (example value "v1beta") for client libraries that create different artifact per version.

Reply:
HI @suztomo, the path to the proto file is the same as the API_ID (as described in the environment variable, i.e., the unique identifier). For example, for google cloud it is google.cloud.kms.v1. Given that different languages may want to do different things to that path, I'll leave it as is.

Reply:
By any chance, are the environment variable API_ID and ApiField's apiId different?

If they are different, I feel it's confusing. How might we avoid introducing the confusion?